src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -106,7 +113,7 @@ public void evaluate() throws Throwable {
                     base.evaluate();
                 }
 
-                private List<PluginSpec> combinePlugins(WithPlugins... wp) {
+                private List<PluginSpec> combinePlugins(List<WithPlugins> wp) {
```

Comment:
@jtnord if I am not wrong, that is already handled by the plugins installation logic. It would be the same as if someone put the same plugin twice in the annotation.